src/lib/transaction_snark_scan_state/transaction_snark_scan_state.ml

```diff
@@ -97,18 +122,38 @@ end = struct
     Ledger_proof_with_sok_message.t Parallel_scan.State.Completed_job.t
   [@@deriving sexp, bin_io]
```

Comment:
Can you include this as well? It is for the other type parameter

Reply:
Done.